rbe.bazelrc

```diff
@@ -0,0 +1,73 @@
+# Copyright 2016 The Bazel Authors. All rights reserved.
```

Comment:
Should our version still use the Bazel copyright header, or could we update this?

Reply:
Yeah I have no idea... It seems weird to copy the whole file and change the header, but OTOH I've already made some edits and we're going to make more edits. I think practically it probably doesn't matter too much. I guess I could read the Apache license and see what it actually says about copying...

But this is all irrelevant because I just inlined it, at which point I'm definitely not adding another license header